cache/lru_cache_test.go

```diff
@@ -1,26 +1,29 @@
 // Copyright (C) 2019-2024, Ava Labs, Inc. All rights reserved.
 // See the file LICENSE for licensing terms.
 
-package cache
+package cache_test
 
 import (
 	"testing"
 
 	"github.com/stretchr/testify/require"
 
+	"github.com/ava-labs/avalanchego/cache/cachetest"
 	"github.com/ava-labs/avalanchego/ids"
+
+	. "github.com/ava-labs/avalanchego/cache"
```

Comment:
I've kind of hated dot imports because I like being able to see the dependency between packages... But I could be convinced of its usefulness (this would be the first time we've used it in the repo afaik)

Reply:
I was going to document this before sending review. One of the few times I think it's acceptable is when package `foo_test` is testing package `foo` (i.e. they're in the same directory and the former is effectively a regular test) and you don't want to add `foo.` in front of all identifiers.

That said, I'm happy to go back to including the fully qualified identifiers, but will punt that to the end of the PR.